shared/fs/index.js

```diff
@@ -66,10 +52,10 @@ const FileRow = RowConnector(({path, name, icon, onOpen}: FileRowProps) => (
 
 const rowPlaceholderIcon = isMobile ? 'icon-folder-private-24' : 'icon-folder-private-24'
 const placeholderTextStyle = {
-  width: '256px',
   backgroundColor: 'lightGrey',
   height: '16px',
   marginTop: '4px',
+  width: '256px',
```

Comment:
don't add px. its not compatible with native and react will add it for you automatically (except lineheight)